clang/lib/Driver/Driver.cpp

```diff
@@ -4714,10 +4714,9 @@ class OffloadingActionBuilder final {
       int NumOfDeviceLibLinked = 0;
       // Currently, all SYCL device libraries will be linked by default. Linkage
       // of "internal" libraries cannot be affected via -fno-sycl-device-lib.
-      llvm::StringMap<bool> devicelib_link_info = {{"libc", true},
-                                                   {"libm-fp32", true},
-                                                   {"libm-fp64", true},
-                                                   {"internal", true}};
+      llvm::StringMap<bool> devicelib_link_info = {
+          {"libc", true},        {"libm-fp32", true},   {"libm-fp64", true},
+          {"libimf-fp32", true}, {"libimf-fp64", true}, {"internal", true}};
```

Comment:
Could you add tests verifying the addition of the new devicelibs during compilation?

Reply:
Hi, @mdtoguchi 
The test for imf devicelib have been already added. Could you review again?

Thanks very much.